### ISSUE TITLE
General Usage + Validation Data Usage + Batch Size

### ISSUE THREAD

**Issue by armlesscoder98** [philipperemy/cond_rnn#54]
1) In the general behavior of fitting an RNN model, we are using validation data to see the loss changes on it, too! Is it possible to apply it in Conditional Recurrent? I couldn't see it in the examples. 

history = model.fit(X_train, y_train, epochs=10, batch_size=8, **validation_data=(X_test, y_test)**, verbose=0)

2) My other question is if we use batch_size as an equal number while fitting (like in the above), does it also take that number of batch size for the external data also?

3) I haven't applied yet Conditional Recurrent; I also have a medium level of knowledge of coding, so if my question becomes too silly, please understand me and reply in that way. I'm just asking this to be sure of what I'm doing. Let's assume that I have input data of 3 cities' temperature values and I also have additional data like given below. While I want to make a prediction of the temperature of one of the cities, I also want to use additional data, but I don't want to look back too much data in the additional data because it causes noise. For this reason, I just want to go back for 3 days for the additional data; however, in my original data, I want to create samples while looking back 30 days to make a prediction. This Conditional Recurrent library provides me with that, am I right?
![aa](https://github.com/philipperemy/cond_rnn/assets/140641717/856a40b1-e306-492a-b35a-37dfd43c5f3b)
